pallets/runtime/src/multisig.rs

```diff
@@ -71,26 +71,26 @@ decl_storage! {
         /// Nonce to ensure unique MultiSig addresses are generated. starts from 1.
         pub MultiSigNonce get(fn ms_nonce) build(|_| 1u64): u64;
         /// Signers of a multisig. (mulisig, signer) => signer.
-        pub MultiSigSigners: double_map hasher(blake2_256) T::AccountId, hasher(blake2_256) Signatory => Signatory;
+        pub MultiSigSigners: double_map hasher(twox_64_concat) T::AccountId, hasher(blake2_128_concat) Signatory => Signatory;
         /// Number of approved/accepted signers of a multisig.
-        pub NumberOfSigners get(fn number_of_signers): map hasher(blake2_256) T::AccountId => u64;
+        pub NumberOfSigners get(fn number_of_signers): map hasher(twox_64_concat) T::AccountId => u64;
         /// Confirmations required before processing a multisig tx
-        pub MultiSigSignsRequired get(fn ms_signs_required): map hasher(blake2_256) T::AccountId => u64;
+        pub MultiSigSignsRequired get(fn ms_signs_required): map hasher(twox_64_concat) T::AccountId => u64;
         /// Number of transactions proposed in a multisig. Used as tx id. starts from 0
-        pub MultiSigTxDone get(fn ms_tx_done): map hasher(blake2_256) T::AccountId => u64;
+        pub MultiSigTxDone get(fn ms_tx_done): map hasher(twox_64_concat) T::AccountId => u64;
         /// Proposals presented for voting to a multisig (multisig, proposal id) => Option<proposal>.
-        pub Proposals get(fn proposals): map hasher(blake2_256) (T::AccountId, u64) => Option<T::Proposal>;
+        pub Proposals get(fn proposals): map hasher(twox_64_concat) (T::AccountId, u64) => Option<T::Proposal>;
         /// A mapping of proposals to their IDs.
         pub ProposalIds get(fn proposal_ids):
-            double_map hasher(blake2_256) T::AccountId, hasher(blake2_256) T::Proposal => Option<u64>;
+            double_map hasher(twox_64_concat) T::AccountId, hasher(blake2_256) T::Proposal => Option<u64>;
```

Comment:
need the change `blake2_256` with `blake2_128_concat`

Reply:
I intentionally left this as `blake2_256` because `T::Proposal` will be relatively big in size so it doesn't make sense to concatenate it here when we can fetch it from a different mapping anyway.